src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -4034,6 +4046,30 @@ const std::string ApplicationManagerImpl::DirectoryTypeToString(
   return "Unknown";
 }
 
+bool ApplicationManagerImpl::AddLockScreenDismissalWarningToMessage(
```

Comment:
@IGapchuk I feel it is a good method for message helper. And from there it can be used both in app_mng and command

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6

Reply:
@IGapchuk Still not in message helper. Do you have Any concerns about it?